ddtrace/profile/exporter/http.py

```diff
@@ -0,0 +1,187 @@
+            + b"\r\n"
+            b'Content-Disposition: form-data; name="chunk-data"; filename="profile.pb.gz"\r\n'
+            + b"Content-Type: application/octet-stream\r\n\r\n"
+            + fields["chunk-data"]
+            + b"\r\n--%s--\r\n" % boundary
+        )
+
+        content_type = b"multipart/form-data; boundary=%s" % boundary
+
+        return content_type, body
+
+    @staticmethod
+    def _get_tags(service):
+        tags = {
+            "service": service.encode("utf-8"),
+            "host": HOSTNAME.encode("utf-8"),
+            "runtime-id": RUNTIME_ID,
+            "language": b"python",
+            "runtime": PYTHON_IMPLEMENTATION,
+            "runtime_version": PYTHON_VERSION,
+            "profiler_version": ddtrace.__version__.encode("utf-8"),
+        }
+        user_tags = os.getenv("DD_PROFILING_TAGS")
+        if user_tags:
+            for tag in user_tags.split(","):
+                try:
+                    key, value = tag.split(":", 1)
+                except ValueError:
+                    LOG.error("Malformed tag in DD_PROFILING_TAGS: %s", tag)
+                else:
+                    if isinstance(value, six.text_type):
+                        value = value.encode("utf-8")
+                    tags[key] = value
+        return tags
+
+    def export(self, events, pid=None):
+        """Export events to an HTTP endpoint.
+
+        :param events: The event dictionary from a `ddtrace.profile.recorder.Recorder`.
+        :param pid: Filter events by process ID. Default is current PID. Use `False` to disable filtering.
+        """
+        if not self.endpoint:
+            raise InvalidEndpoint("Endpoint is empty")
+        parsed = urlparse.urlparse(self.endpoint)
+        if parsed.scheme == "https":
+            client_class = http_client.HTTPSConnection
+        else:
+            client_class = http_client.HTTPConnection
+        if ":" in parsed.netloc:
+            host, port = parsed.netloc.split(":", 1)
+        else:
+            host, port = parsed.netloc, None
+        client = client_class(host, port, timeout=self.timeout)
+
+        common_headers = {
+            "DD-API-KEY": self.api_key.encode(),
+        }
+
+        exceptions = []
+        profile = super(PprofHTTPExporter, self).export(events)
+        s = six.BytesIO()
+        with gzip.GzipFile(fileobj=s, mode="wb") as gz:
+            gz.write(profile.SerializeToString())
+        fields = {
+            "runtime-id": RUNTIME_ID,
+            "recording-start": (
+                datetime.datetime.utcfromtimestamp(profile.time_nanos // 10e8).isoformat() + "Z"
+            ).encode(),
+            "recording-end": (datetime.datetime.utcnow().replace(microsecond=0).isoformat() + "Z").encode(),
+            "runtime": PYTHON_IMPLEMENTATION,
+            "format": b"pprof",
+            "type": b"cpu+alloc+exceptions",
+            "chunk-data": s.getvalue(),
+        }
+        if "DD_SERVICE_NAME" in os.environ:
```

Comment:
Sorry to throw a wrench into the complexity here, but we need to move to `DD_SERVICE`, but will probably need to support `{DD,DATADOG}_SERVICE_NAME` for backwards compatibility